community/consistency-check/src/main/java/org/neo4j/consistency/checking/SchemaRecordCheck.java

```diff
@@ -155,7 +143,10 @@ private void checkUniquenessConstraintRule( UniquePropertyConstraintRule rule,
     {
         if ( phase == Phase.CHECK_RULES )
         {
+            engine.comparativeCheck( records.label( rule.getLabel() ), VALID_LABEL );
```

Comment:
Next 3 lines of code used couple of times and can be extracted to separate method